modules/ROOT/pages/partner-manager-delete-partner.adoc

```diff
@@ -2,7 +2,7 @@
 
 If you are no longer trading with a partner, you can delete the partner.
 
-When you delete a partner, the partner is removed from the Partner Manager's *Partners* page, and all the partner's data is deleted, except for activities for the partner that occurred before the partner was deleted.
+When you delete a partner, the partner is removed from the Partner Manager's *Partners* page. All of the partner's data is deleted, except for partner activities that occurred before the partner was deleted.
```

Comment:
`Partner Manager *Partners* page.`

Reply:
The actual page name is the name of a partner, and there are often multiple partners. So I don't put it in bold.